Hyperswitch-React-Demo-App/promptScript.js

```diff
@@ -1,22 +1,15 @@
 const fs = require("fs");
 const prompt = require("prompt-sync")({ sigint: true });
-const publishableKey = prompt("Publishable Key : ");
-const secretKey = prompt("Secret Key : ");
-const serverURL = prompt("Self-hosted Hyperswitch Server URL : ");
-const clientURL = prompt("Self-hosted Hyperswitch Client URL : ");
-
-const appServerURL = prompt("Application Server URL : ");
-const appClientURL = prompt("Application Client URL : ");
 
 const envPath = "./.env";
 
 const obj = {
-  HYPERSWITCH_PUBLISHABLE_KEY: publishableKey,
-  HYPERSWITCH_SECRET_KEY: secretKey,
-  HYPERSWITCH_SERVER_URL: serverURL,
-  HYPERSWITCH_CLIENT_URL: clientURL,
-  SELF_SERVER_URL: appServerURL,
-  SELF_CLIENT_URL: appClientURL,
+  HYPERSWITCH_PUBLISHABLE_KEY: "Publishable Key",
```

Comment:
if you are having prompts only when the env is empty you might as well initialize these to empty string 
just to make sure that script doesnt update the value with these

Reply:
These values are not what will be added in the env. These are the string for the prompt messages. I have moved the prompt inside the initializeValues function

Reply:
got it